### PR TITLE
BigAnimal: Control status remediation

### PR DESCRIPTION
**Pull request by moiznalwalla**

### CODE REVIEW

File: product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Remediating AWS control findings"
+
+---
+
+This page describes the control findings in your AWS Security Hub and suggests potential remediation steps for your BigAnimal resources.
```
Comment (l0r3bo) [inline product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx:6]: I would remove the term "control" which is of very widespread use (we tend to use it a lot internally, I may have used it myself when discussing this), but difficult interpretation. Moreover it is not used at all in the security hub documentation, which only talks about "findings".

Suggested rewording: "This page describes findings in your AWS Security Hub...".

Whether we want to add also "that may relate to BigAnimal cloud resources" is an interesting point, I think we should if that's not clear/implied by other pieces of documentation. And especially we should be consistent with what is discussed on the Azure side.

File: product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+title: "Remediating AWS control findings"
+
+---
+
+This page describes the control findings in your AWS Security Hub and suggests potential remediation steps for your BigAnimal resources. 
+
+!!! Important
+    Remediation steps provided below might increase your cloud costs. Before proceeding, make sure that you evaluate and prioritize the suggested actions based on your organization policies. 
+
+The control findings are listed below based on the level of impact to your resources.
```
Comment (l0r3bo) [inline product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx:11]: Same remark as above on the usage of "control". Suggested rewording "The findings...".

Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx]: incorporated your feedback

File: product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx (hunk 1 above)
Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx]: Incorporated your feedback.

**Comment by drothery-edb** [comment]
obsolete

**Pull request closed without merging**